content/en/blog/_posts/2020-05-xx-third-party-dual-sourced-content.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "How Docs Handle Third Party and Dual Sourced Content"
+date: 2020-05-xx
+slug: third-party-dual-sourced
+url: /blog/2020/05/third-party-dual-sourced-content
+---
+
+**Author:** Zach Corleissen | [GitHub](https://github.com/zacharysarah/) | [Twitter](https://twitter.com/zachorsarah)
+
+*Editor's note: Zach is one of the chairs for the Kubernetes documentation special interest group (SIG Docs).*
+
+Late last summer, SIG Docs started a community conversation about third party content in Kubernetes docs. This conversation became a [Kubernetes Enhancement Proposal](https://github.com/kubernetes/enhancements/pull/1327) (KEP) and, after five months for review and comment, SIG Architecture approved the KEP as a [content guide](/docs/contribute/style/content-guide/) for Kubernetes docs.
+
+Here's how Kubernetes docs handle third party content now:
+
+> Links to active content in the Kubernetes project (projects in the kubernetes and kubernetes-sigs GitHub orgs) are always allowed.
+>
+> Kubernetes requires some third party content to function. Examples include container runtimes (containerd, CRI-O, Docker), networking policy (CNI plugins), Ingress controllers, and logging.
+>
+> Docs can link to third-party open source software (OSS) outside the Kubernetes project if it’s necessary for Kubernetes to function.
+
+These common-sense guidelines make sure that Kubernetes docs document Kubernetes.
+
```

Comment:
nit: common sense?
nit: These common sense guidelines make sure that Kubernetes docs [fairly|wholly|fully] document Kubernetes.